spec/requests/service_specific_information_spec.rb

```diff
@@ -16,10 +16,10 @@
     end
 
     context 'when service specific posts exist' do
-      let!(:branch) { create(:branch_of_service, name: 'Air Force') }
+      let!(:branch) { create(:branch_of_service, name: 'Navy') }
```

Comment:
Easier to just pick a page that doesn't have a video on it for the tests, I take it?

Reply:
exactly